packages/block-editor/src/components/block-list/style.scss

```diff
@@ -424,9 +468,9 @@
 
 	// Full-wide
 	&[data-align="full"] {
-		// Position hover label on the right for the top level block.
+		// Position hover label on the left for the top level block.
```

Comment:
Is it on the left, or on the right?

There are still existing styles which assume right-placement for the wide/full hover label:

https://github.com/WordPress/gutenberg/blob/e209cfb77469b0e022198f75ab1fd641f98b229a/packages/block-editor/src/components/block-list/style.scss#L427-L430

I'm supposing that left placement is reasonable only in part due to the regression of #14817 

Fixing #14817 introduces an undesirable overlap:

![image](https://user-images.githubusercontent.com/1779930/56296973-78220a80-60fd-11e9-9b4c-23505a32ebc8.png)

Reply:
> Fixing #14817 introduces an undesirable overlap:

Follow-up: #15022 (proposes moving to the right)